analyzers/tests/SonarAnalyzer.UnitTest/Rules/WeakSslTlsProtocolsTest.cs

```diff
@@ -32,20 +29,18 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class WeakSslTlsProtocolsTest
     {
+        private readonly VerifierBuilder builderCS = WithReferences(new VerifierBuilder<CS.WeakSslTlsProtocols>());
+        private readonly VerifierBuilder builderVB = WithReferences(new VerifierBuilder<VB.WeakSslTlsProtocols>());
+
         [TestMethod]
         public void WeakSslTlsProtocols_CSharp() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\WeakSslTlsProtocols.cs",
-                                    new CS.WeakSslTlsProtocols(),
-                                    ParseOptionsHelper.FromCSharp8,
-                                    GetAdditionalReferences());
+            builderCS.AddPaths("WeakSslTlsProtocols.cs").WithOptions(ParseOptionsHelper.FromCSharp8).Verify();
```

Comment:
The syntax is on the edge this time, both options are good. I've chosen fields because it makes the code more table-like and therefore easier to read and see what the actual difference is.